audio-worklet/noisegate-test.html

```diff
@@ -18,12 +18,11 @@
     <script src="test/audit-util.js"></script>
     <script src="test/audit.js"></script>
     <script src="src/noisegate-script-processor.js"></script>
-
     <script id="noisegate-test-code">
       // Sample rate is a power of two to avoid interpolation from delay node.
       // It is held artificially low for easy inspection of audio buffers.
       // All tests pass for all legal powers in the range [3000, 384000].
-      const sampleRate = Math.pow(2, 12);
+      const sampleRate = Math.pow(2,16);
```

Comment:
Can we just right 65536 instead of 2^16?

This also contradicts line 23 above.